internal/datanode/channel_meta.go

```diff
@@ -48,6 +50,12 @@ var (
 	newVarCharPrimaryKey = storage.NewVarCharPrimaryKey
 )
 
+type syncFlag struct {
+	needToSync *atomic.Bool
+	// syncKnown indicates sync is being executed, to prevent frequently syncing
+	syncKnown *atomic.Bool
```

Comment:
syncKnown may not be necessary?
we can check all needToSync flag and see if there is anotjer sync?

Reply:
Without syncKnown, sync policy would sync frequenly (per 200ms)?